ziti-fabric/subcmd/snapshot_db.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
why is this in fabric ? i'd not have thought to look for it in ziti-fabric

Reply:
It needs to be in both places. It's fabric functionality, so it needs to be in ziti-fabric, but it should be available via the REST API as well. I haven't added it to the edge yet, not sure if I need to or not. When the fabric management API gets consolidated into REST, it'll be available there. I'll have to ask if ziti-fabric is sufficient for now

Reply:
Went ahead and added it to edge so it's available in both now